yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -217,13 +220,17 @@ export function makeAccumulatedData(seed = 1): CombinedAccumulatedData {
     makeAggregationObject(seed),
     makeTuple(KERNEL_NEW_COMMITMENTS_LENGTH, fr, seed + 0x100),
     makeTuple(KERNEL_NEW_NULLIFIERS_LENGTH, fr, seed + 0x200),
-    makeTuple(KERNEL_PRIVATE_CALL_STACK_LENGTH, fr, seed + 0x300),
+    makeTuple(KERNEL_PRIVATE_CALL_STACK_LENGTH, Fr.zero), // private call stack must be empty
```

Comment:
Curious why this only popped up now

Reply:
I don't remember doing the change and I think it's a result of incorrectly perfomed rebase. I should have better check the PR's diff. Addressed in [e1aae84](https://github.com/AztecProtocol/aztec-packages/pull/636/commits/e1aae8404fa0d9d8546c75617e2b9f0702199f62). Thanks for catching that.